app/models/region.rb

```diff
@@ -0,0 +1,5 @@
+class Region < ApplicationRecord
```

Comment:
Is this being used/populated somewhere?

Reply:
@Levia I believe this needs to be there for FactoryBot to be able to create a `country` as a `region` is required.